### DISCUSSION TITLE
Sakura Succubus 1,2,3,4,5

### DISCUSSION THREAD

**Discussion by GameTester1** [skyline-emu/skyline#1485] [Game Requests]
Devs, plz fix it sakura succubus🙏😘. I hope next update can be better. Good job devs
![Screenshot_2022-06-06-10-46-54-798_skyline emu](https://user-images.githubusercontent.com/106941200/172086048-123a56da-074e-44f2-8d45-9e256eb85412.jpg)
![Screenshot_2022-06-06-10-47-01-376_skyline emu](https://user-images.githubusercontent.com/106941200/172086069-d156b898-0b58-42ea-8be6-c83bd0295acd.jpg)

**Comment by thekillerwave** [comment]
down bad
